src/pages/admin/reports/components/program-expenses-section/components/program-expenses-table/ProgramExpensesTable.module.scss

```
@@ -207,3 +207,91 @@
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    height: 80px;
+    overflow: visible;
+    position: relative;
+}
+
+.category-edit-select {
+    width: 240px;
+
+    :global(.select-head) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)